plugin/hooks.go

```diff
@@ -149,4 +153,16 @@ type Hooks interface {
 	// Note that this method will be called for files uploaded by plugins, including the plugin that uploaded the post.
 	// FileInfo.Size will be automatically set properly if you modify the file.
 	FileWillBeUploaded(c *Context, info *model.FileInfo, file io.Reader, output io.Writer) (*model.FileInfo, string)
+
+	// PushNotificationWillBeSent is invoked before the push notification has been sent to the push proxy server.
```

Comment:
Could you please document the minimum server version:
```suggestion
	// PushNotificationWillBeSent is invoked before the push notification has been sent to the push proxy server.
	// Minimum server version: 5.10
```

Reply:
Done

Reply:
Done